src/isar/services/readers/map_reader.py

```diff
@@ -37,13 +35,14 @@ def get_predefined_map_configs(self) -> Optional[dict]:
         json_files = self.predefined_map_config_folder.glob("*.json")
         for file in json_files:
             path_to_file = self.predefined_map_config_folder.joinpath(file.name)
-            map_config: MapConfig = self.get_map_config(path_to_file)
-            if map_config is None:
+            try:
+                map_config: MapConfig = self.get_map_config(path_to_file)
+            except BaseReaderError:
```

Comment:
Can we rather let this fail instead of catching?

Reply:
Same as the above, do we want it to still run with the remaining map configs if we added a faulty new one?

Reply:
I would want to fail, I think. This is your decision. You could ask someone else as well if you are not sure. I can jjoin that discussion if wanted.